src/game/server/player.cpp

```diff
@@ -2631,6 +2632,17 @@ void CBasePlayer::ObserverUse( bool bIsPressed )
 
 void CBasePlayer::JumptoPosition(const Vector &origin, const QAngle &angles)
 {
+	Vector neworigin;
+	QAngle newangles;
+
+	// Clamp the position and angles to prevent crashes
+	neworigin.x = clamp( origin.x, MIN_COORD_FLOAT, MAX_COORD_FLOAT );
```

Comment:
These lines don't do anything, the only case where this validation is run is when switching spectator targets